en-US/emails/2017/FxnU_oct.lang

```diff
@@ -0,0 +1,146 @@
+Not quite. Firefox is on a mission — to be faster and leaner than ever, always respectful of your privacy and open like no other browser out there. Because it’s built for people, not profit. Firefox is part of Mozilla’s mission to keep you in control of your online experience and ensure that the web stays weird, healthy and inviting to all.
+
+
+;When you browse with Firefox, you’re supporting a nonprofit organization that fights for net neutrality, promotes sensible copyright reform in the EU, teaches web literacy around the world and keeps a close eye on the overall health of the internet.
+When you browse with Firefox, you’re supporting a nonprofit organization that fights for net neutrality, promotes sensible copyright reform in the EU, teaches web literacy around the world and keeps a close eye on the overall health of the internet.
+
+
+;We’re here to help you go your own way online and give you the power to browse freely, without being tracked or profited from.
+We’re here to help you go your own way online and give you the power to browse freely, without being tracked or profited from.
+
+
+;Because you are not a product. You’re an explorer on the open web. You deserve something different. Something better.
+Because you are not a product. You’re an explorer on the open web. You deserve something different. Something better.
+
+
+;We’ve got your back, <br> The Firefox Team
+We’ve got your back, <br> The Firefox Team
+
+
+;P.S. Questions? Drop us a line.
+P.S. Questions? Drop us a line.
+
+
+# Email 3
+# Subject Line
+;Your privacy = your business
+Your privacy = your business
+
+
+# Preheader
+;Take control of your privacy with Firefox.
+Take control of your privacy with Firefox.
+
+
+# Headline
+;Keep trackers off your trail
+Keep trackers off your trail
+
+
+;You shouldn’t have to explore the web with someone looking over your shoulder, but most websites use tracking technology that doesn’t give you any say in the matter.
+You shouldn’t have to explore the web with someone looking over your shoulder, but most websites use tracking technology that doesn’t give you any say in the matter.
+
+
+;Now you can take control. Use Private Browsing to give online snoopers the boot. And choose what you share with Firefox privacy extensions.
+Now you can take control. Use Private Browsing to give online snoopers the boot. And choose what you share with Firefox privacy extensions.
+
+
+;You can even take your privacy with you. Firefox Focus for Android and iOS and make your digital footprint disappear with a single tap.
+You can even take your privacy with you. Firefox Focus for Android and iOS and make your digital footprint disappear with a single tap.
+
+
+;It’s a wild web out there — explore it with confidence.
+It’s a wild web out there — explore it with confidence.
+
+
+;Safe travels, <br> The Firefox Team
+Safe travels, <br> The Firefox Team
```

Comment:
Remove the spaces around <br>, you don't want the new line starting with a leading space.